app/src/handlers/census/CensusProxy.ts

```diff
@@ -98,10 +117,9 @@ export default class CensusProxy {
             case 'ContinentLock':
                 this.continentLockHandler.handle(event);
                 break;
-            // Documented but never happens
-            // case 'ContinentUnlock':
-            //     eventStore.storeContinentUnlock(payload);
-            //     break;
+            case 'ContinentUnlock':
```

Comment:
Still doesn't happen, how come you uncommented?

Reply:
The problem was that ContinentUnlock was not part of GenericEvent.
Therefore I implemented the handler but was not able to call it.